packages/react-components/review/beta/react-components.api.md

```diff
@@ -228,12 +228,33 @@ export interface _CaptionsBannerProps {
     onRenderAvatar?: OnRenderAvatarCallback;
 }
 
+// @beta
+export interface ChatAttachment {
+    // (undocumented)
+    attachmentType: string;
+    // (undocumented)
+    id: string;
+    // (undocumented)
+    name?: string;
+    // (undocumented)
+    previewUrl?: string;
+    // (undocumented)
+    status: ChatAttachmentStatus;
+    // (undocumented)
+    url: string;
+}
+
+// @beta
+export type ChatAttachmentStatus = 'downloading' | 'completed' | 'failed' | 'none';
```

Comment:
Hi, how is this status being used? 

for inline images, would it be status for downloading preview image or original image?

for file sharing that won't have a download status, would it always be 'none'?

cc @palatter

Reply:
This is a good question. I think for now the thought was to use it for the preview url, since that is all we are currently supporting. But I think it might make sense to have 2, one for the preview url and one for the url. 
Why would file sharing not have any status?

Reply:
file sharing would only have a URL which would navigate user to a web page where they need to enter password manually. There's no direct download available due to limitations on AAD

Reply:
also can you clarify why we need to know status? Are you thinking more like a long term goal for graph team use cases?

Reply:
Is this to only be used for inline images? if this is also a update to the existing file sharing API there is a progress indicator already that contoso can use I believe to display the upload percentage/ if it fails. (my recollection of this is quite bad so just take it as my 2 cents here)

Reply:
yea I would vote to remove it if we only use it for future. We might not want Contoso getting confused by what this status is for (preview download vs. full-res download)